data_internet.go

```diff
@@ -2,6 +2,8 @@ package gofakeit
 
 // DataInternet consists of various internet information
 var DataInternet = map[string][]string{
-	"browser":       {"firefox", "chrome", "internetExplorer", "opera", "safari"},
-	"domain_suffix": {"com", "biz", "info", "name", "net", "org"},
+	"browser":           {"firefox", "chrome", "internetExplorer", "opera", "safari"},
```

Comment:
I think "ie" might be a better representation of the browser. Might want to also add "edge",

Reply:
Haha i just realized this was mine. if you could make this change I would be appreciated. Otherwise i can do it myself.

Reply:
Should I make an another PR for resolving that? Because this PR is for just HTTP methods. How do you think?